typescript/src/events/guilds/members/guildMemberUpdateRolesNotify.ts

```diff
@@ -11,9 +10,9 @@ import type { TFunction } from 'i18next';
 @ApplyOptions<EventOptions>({ event: Events.GuildMemberUpdate })
 export class UserEvent extends Event {
 	public async run(previous: GuildMember, next: GuildMember) {
-		const [enabled, t] = await next.guild.readSettings((settings) => [settings[GuildSettings.Events.MemberRoleUpdate], settings.getLanguage()]);
-
-		if (!enabled) return;
+		const key = GuildSettings.Channels.Logs.MemberRoleUpdate;
+		const [channelId, t] = await next.guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

Comment:
```suggestion
		const [logChannelId, t] = await next.guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.